core/src/Streamly/Internal/Data/Stream/Bottom.hs

```diff
@@ -488,7 +488,7 @@ drop n m = fromStreamD $ D.drop n $ toStreamD m
 -- | Find all the indices where the element in the stream satisfies the given
 -- predicate.
 --
--- >>> findIndices p = Stream.fold (Fold.findIndices p)
+-- >>> findIndices p = Stream.scanMaybe (Fold.findIndices p)
```

Comment:
Maybe you can also add an example doc spec here so we don't make these kinds of mistakes again.